src/lib/utils.ts

```
@@ -4,3 +4,59 @@ import { twMerge } from "tailwind-merge"
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs))
 }
+
+// Simple date formatting utility 
+export function formatDate(timestamp: number) {
+  // Validate input
+  if (typeof timestamp !== 'number' || isNaN(timestamp) || !isFinite(timestamp)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)